### ISSUE TITLE
Missing metallitic and roughness map for screen space reflection?

### ISSUE THREAD

**Issue by Pannic17** [mrdoob/three.js#23247]
When using ssr pass in postprocessing, I noticed whatever the object added to the scene, the reflection of each mesh renders either very high or just none. Is there any ways or support for applying metalness map and roughness map to the ssr pass for pbr?

**Comment by gonnavis** [comment]
Hello @Pannic17 . I tried it, but pbr integration is currently too difficult for me, I need to have a deeper understanding of pbr to complete this feature.
However, at present this new pr can at least adjust the intensity of each object separately, but it has not been merged. If necessary, you can merge it yourself locally.
https://github.com/mrdoob/three.js/pull/21487#issuecomment-837410001

**Comment by Pannic17** [comment]
Thanks @gonnavis, your code helped a lot. Base on your new pr and I just found that in SSRShader, the wrong channel of metalness map was used to generate ssr. 
`main(){ float metalness=texture2D(tMetalness,vUv).r;`
In main of fragment shader, the red channel of metalness map was used, which should be the blue channel.
`float metalness=texture2D(tMetalness,vUv).b;`
After I corrected this, it seems ssr generated according to the correct metalness map, however, the roughnes still remain unaffected. Looking forward for solution for that
